doc/developer-guide.rst

```diff
@@ -32,6 +32,7 @@ A Motor framework is a module implementing these properties and functions:
 - ``coroutine`` (**DEPRECATED**)
 - ``get_event_loop``
 - ``get_future``
+- ``create_task``
```

Comment:
This can be added because https://www.tornadoweb.org/en/branch5.1/_modules/tornado/platform/asyncio.html#AsyncIOMainLoop offers access to the underlying `asyncio` event loop and starting in this version we only support Python and Tornado versions where this API will work.